tests/components/trafikverket_ferry/conftest.py

```diff
@@ -0,0 +1,80 @@
+"""Fixtures for Trafikverket Ferry integration tests."""
+from __future__ import annotations
+
+from datetime import datetime, timedelta
+from unittest.mock import patch
+
+import pytest
+from pytrafikverket.trafikverket_ferry import FerryStop
+
+from homeassistant.components.trafikverket_ferry.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+from homeassistant.util import dt
+
+from . import ENTRY_CONFIG
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture()
+async def load_int(
+    hass: HomeAssistant, get_ferries: list[FerryStop]
+) -> MockConfigEntry:
+    """Set up the Sensibo integration in Home Assistant."""
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        source=SOURCE_USER,
+        data=ENTRY_CONFIG,
+        entry_id="1",
+        unique_id="123",
+    )
+
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.trafikverket_ferry.coordinator.TrafikverketFerry.async_get_next_ferry_stops",
+        return_value=get_ferries,
+    ):
+        await hass.config_entries.async_setup(config_entry.entry_id)
+        await hass.async_block_till_done()
+
+    return config_entry
```

Comment:
If the patch should be in effect throughout the test, you can do something like this:
```suggestion
        await hass.async_block_till_done()
        yield config_entry
```